openstudiocore/ruby/openstudio/sketchup_plugin/user_scripts/Reports/OSM_Diagnostic_Script.rb

```diff
@@ -451,6 +451,32 @@ def run(runner, user_arguments)
     else
       puts "no orphaned PortList objects were found"
     end
+  
+    # find and remove orphan LifeCycleCost objects
+    puts ""
+    puts "Removing LifeCycleCost objects that are not connected to any model object"
+    #get all LifeCycleCost objects in the model
+    lcc_objects = model.getObjectsByType("OS:LifeCycleCost".to_IddObjectType)
+    #make an array to store the names of the orphan LifeCycleCost objects
+    orphaned_lcc_objects = Array.new
+    #loop through all LifeCycleCost objects, checking for missing Item Name
+    lcc_objects.each do |lcc_object|
+      if lcc_object.isEmpty(4)
```

Comment:
@macumber is there a reason to refer to field by position 4 instead of calling lcc_object.item